my-app/src/App.css

```diff
@@ -0,0 +1,112 @@
+* {
+  font-family: "Lato", sans-serif;
+  margin: 0;
+  padding: 0;
+  /* font-size: 12px; */
```

Comment:
Committing source code that is commented out without an explanation is usually not a great idea (as it can become confusing rather quickly).

If this exists for a purpose, it's better to accompany it with an explanation. This could just be a quick `// TODO` style comment

Reply:
Understood, will add comments with `// TODO`. Thanks!